sirepo/pkcli/job_agent.py

```diff
@@ -273,26 +275,25 @@ def _fastcgi_accept(self, connection, *args, **kwargs):
                 )
             except Exception as e:
                 pkdlog('msg={} error={} stack={}', msg, e, pkdexc())
-
-        pkdlog('msg={} error={} stack={}', msg, error, stack)
-        # destroy _fastcgi state first, then send replies to avoid
-        # asynchronous modification of _fastcgi state.
-        self._fastcgi_remove_handler()
-        q = self._fastcgi_msg_q
-        self._fastcgi_msg_q = None
-        self._fastcgi_cmd.destroy()
-        self._fastcgi_cmd = None
-        if msg:
-            await _reply_error(msg)
-        while q.qsize() > 0:
-            await _reply_error(q.get_nowait())
-            q.task_done()
+        # If self.fastcgi_cmd is None we initiated the kill so not an error
+        if self.fastcgi_cmd:
```

Comment:
```py
if not self.fastcgi_cmd:
     return
```

I think we still want to log. 

Maybe put the logging back at the `except` and only call the handler if fastcgi_cmd?

Reply:
Ya that's clearer.